### ISSUE TITLE
Add checking for existence of rainlab translate tables

### ISSUE THREAD

**Issue by cyril-design** [rainlab/translate-plugin#718]
Could the plugin check for the existence of tables ?
(I had problems with Migrate command...)
```php
if (!Schema::hasTable('rainlab_translate_attributes')) {
(...)
``` 
Thanks !

**Comment by daftspunk** [comment]
Hi @cyril-design 

It can be common if plugins have some overlaps. It may be better to introduce a flag to the migrate command for this. Something like `php artisan october:migrate --skip-errors` which will proceed past any exceptions and treat the migration as applied even if it fails.

**Comment by cyril-design** [comment]
Thanks for your reply !

**Comment by daftspunk** [comment]
This will be available in v3.4

```bash
php artisan october:migrate --skip-errors
```
